libmscore/chord.cpp

```diff
@@ -1735,7 +1735,8 @@ void Chord::cmdUpdateNotes(AccidentalState* as)
       // PITCHED_ and PERCUSSION_STAFF can go note by note
 
       if (staffGroup == StaffGroup::STANDARD) {
-            for (Chord* ch : graceNotesBefore()) {
+            const QVector<Chord*> gnb(graceNotesBefore());
```

Comment:
I thought std::vector was the one to use whenever possible?

Reply:
In the new code in `libmscore`, yes. Switching to `std::vector` here would make it desirable to switch the `graceNotesBefore()` return type to `std::vector` which would require more changes not related to this PR. This could certainly be done but it seems to be better for a separate PR.